File: ShopOnline/Asset/Client/js/contactController.js
var contact = {
    init: function () {
        contact.registerEvent();
    },
    registerEvent: function () {
        $('#btnSend').off('click').on('click', function () {
            var name = $('#txtName').val();
            var phone = $('#txtPhone').val();
            var address = $('#txtAddress').val();
            var email = $('#txtEmail').val();
            var content = $('#txtContent').val();
            $.ajax({
                url: "/Contact/Send",
                type: "POST",
                dataType: "json",
                data: {
                    txtName: name,
                    txtPhone: phone,
                    txtAddress: address,
                    txtEmail: email,
                    txtContent: content
                },
                success: function (respon) {
                    if (respon.status == true) {
                        alert("thành công");
                    }
                }
            });
        });
    }
};
contact.init();

File: ShopOnline/Controllers/ContactController.cs
using _Models.DAO;
using _Models.EF;
using ShopOnline.Common;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace ShopOnline.Controllers
{
    public class ContactController : Controller
    {
        // GET: LienHe
        public ActionResult Index()
        {
            TempData["msg"] = MessageBox.Show(thongbao);
            ContactDAO contactDAO = new ContactDAO();
            var contact = contactDAO.ListAll().SingleOrDefault();
            return View(contact);
        }
        private static string thongbao="";
        public JsonResult Send(string txtName,string txtPhone,string txtAddress,string txtEmail,string txtContent)
        {
            ContactDAO contactDAO = new ContactDAO();
            Feedback feedback = new Feedback();
            feedback.Address = txtAddress;
            feedback.Content = txtContent;
            feedback.Email = txtEmail;
            feedback.CreateDate = DateTime.Now;
            feedback.FeedbackName = txtName;
            txtPhone = string.IsNullOrEmpty(txtPhone) ? "0" : txtPhone;
            feedback.Phone = int.Parse(txtPhone);
            var tempstatus = contactDAO.InsertFeedback(feedback);
            thongbao = "Thành công";
            return Json(new
            {
                status = tempstatus
            });
        }
    }
}